package.json

```diff
@@ -15,8 +15,8 @@
     "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
     "openapi:bundle": "bundle-api-spec -i openapi/index.yaml.template -o openapi/index.yaml -V $npm_package_version",
     "pregenerate": "npm run openapi:bundle && rimraf ./generated",
-    "generate": "npm-run-all generate:backend:*",
-    "generate:backend:api-models": "shx rm -rf generated/backend && shx mkdir -p generated/backend && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v7.28.0/api_backend.yaml --out-dir generated/backend",
+    "generate": "npm-run-all generate:*",
+    "generate:definitions": "shx rm -rf generated/definitions && shx mkdir -p generated/definitions && gen-api-models --api-spec ./openapi/index.yaml --no-strict --out-dir ./generated/definitions",
```

Comment:
Since we are using `@pagopa/io-function-commons` definitions, we could skip this generation